src/masoniteorm/connections/MySQLConnection.py

```diff
@@ -69,28 +69,9 @@ def reconnect(self):
     def get_default_query_grammar(cls):
         return MySQLGrammar
 
-    def get_connection_details(self):
-        """This is responsible for standardizing the normal connection
-        details and passing it into the connection.
-
-        This will eventually be unpacked so make sure the keys are the same as the keywords
-        that should pass to your connection method
-        """
-        connection_details = {}
-        connection_details.setdefault("host", self.connection_details.get("host"))
-        connection_details.setdefault("user", self.connection_details.get("user"))
-        connection_details.setdefault(
-            "password", self.connection_details.get("password")
-        )
-        connection_details.setdefault("port", int(self.connection_details.get("port")))
-        connection_details.setdefault("db", self.connection_details.get("database"))
-        connection_details.update(self.connection_details.get("options", {}))
-
-        return connection_details
-
     @classmethod
     def get_database_name(self):
-        return self().get_connection_details().get("db")
+        return self().database
```

Comment:
I don't believe this will work the way you think it will work. This is a class method. class methods pass in the class into the first variable instead of the instance. So in this example this will always return false because you are instantiating a new class with no parameters.

It should just be 

```python
def get_database_name(self):
    return self.database
```

Reply:
Yes, that's what I had done first. Then as I wasn't sure I switch back (both are working maybe because self() returned the already instantiated class ? like a singleton ?).

Anyway I made requested changes !